wiki/Beatmap/Approach_rate/fr.md

```diff
@@ -3,8 +3,6 @@ tags:
   - approach time
   - AR
   - reading
```

Comment:
Maybe we should add `temps d'approche` here ? It's the French translation of `Approach time` on the stable client